deploy/osd-26960/01-cronjob.yaml

```diff
@@ -31,11 +31,10 @@ spec:
 
                   # Path for temporary subscription JSON backup
                   SUB_JSON_PATH="/tmp/sub.json"
-    
-                  # The OLM dance
 
+                  # The OLM dance
                   # Delete CSVs if they exist
-                  CSV_LIST=$(oc get clusterserviceversions.operators.coreos.com -n "$NAMESPACE" | grep "$OPERATOR" || true | awk '{print $1}')
+                  CSV_LIST=$(oc get clusterserviceversions.operators.coreos.com -n "$NAMESPACE" --no-headers | grep "$OPERATOR" | awk '{print $1}' || echo "")
```

Comment:
We use `set -e` so we need to dodge the errors when nothing is found, thus `|| echo ""`